neo/NeoSystem.cs

```diff
@@ -26,7 +26,7 @@ public class NeoSystem : IDisposable
         public IActorRef Consensus { get; private set; }
         public RpcServer RpcServer { get; private set; }
 
-        private readonly Store store;
+        internal readonly Store store;
```

Comment:
In fact, the `Store` used by the `ConsensusService` may not be the `Store` in `NeoSystem`.

Reply:
I see that `StartConsensus()` supports passing a different store. Perhaps both stores should instead be passed when constructing the `NeoSystem` so that it can be accessed. What do you think?

Reply:
I addressed this issue by having the store to use for consensus passed into `NeoSystem`, see the latest changes on this PR.